views/partials/what_to_do.hbs

```diff
@@ -0,0 +1,18 @@
+<ul id="what-to-do">
```

Comment:
Should this be an `<ol>` if we're adding numbers to each item anyways?

Reply:
Oh it's definitely an '<ol>'. What a great, great catch. I'm embarrassed and now thinking back to other times I have misused list tags.

Reply:
whoops, it didn't appreciate my quotation marks evidently...There should be an <ol> there.

Reply:
`<ol>` (I had to have Luke help me).